erp-playbook/Makefile

```diff
@@ -22,6 +22,27 @@ else
 	ansible-playbook -t run_test -l erp-drue -e "{erp_build_number: $(BUILD_NUM)}" -e erp_debian_installer_environment=$(BUILD_ENV) main.yml
 endif
 
+provision-and-run-chase: roles
```

Comment:
Does copy the lines and replace 'drue' with 'chase' look right? Is it possible to use a variable to replace 'drue' and 'chase'?